operators/api/v1alpha2/template_types.go

```diff
@@ -160,6 +160,9 @@ type ContainerStartupOpts struct {
 	ContentPath string `json:"contentPath,omitempty"`
 	// Arguments to be passed to the application container on startup
 	StartupArgs []string `json:"startupArgs,omitempty"`
+	// Whether forcing the container working directory to be the same as the contentPath (or default mydrive path if not specified)
+	// +kubebuilder:default=true
+	EnforceWorkdir bool `json:"enforceWorkdir,omitempty"`
```

Comment:
Do not use a boolean value defaulted to true coupled with the omitempty option, as this has shown to create issues with controllers (a false value is not transmitted, thus considered as true by the backend). In other words, I would suggest removing the omitempty option and adding the `// +kubebuilder:validation:Optional` marker.

Reply:
That's exactly the issue I had in mind when I wrote that, and what I wanted to avoid but, as my usual, I messed up. Thanks

Reply:
My original intent was to leave the omitempty and default it to false instead. What do you think?

Reply:
Would personally use the Optional marker instead of omitempty, regardless of the default value (feel free to select the one you prefer).